components-compose/src/main/java/uk/gov/hmrc/components/compose/atom/button/Button.kt

```diff
@@ -15,4 +15,119 @@
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.sp
+import uk.gov.hmrc.components.compose.ui.theme.HmrcRippleTheme
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme.colors
+import uk.gov.hmrc.components.compose.ui.theme.hmrc_button_size_48
+import uk.gov.hmrc.components.compose.ui.theme.hmrc_icon_size_24
+import uk.gov.hmrc.components.compose.ui.theme.hmrc_spacing_16
+
+@Composable
+fun HmrcButton(
+    text: String,
+    modifier: Modifier = Modifier,
+    textAlign: TextAlign = TextAlign.Center,
+    enabled: Boolean,
+    shape: Shape = RectangleShape,
+    buttonColors: ButtonColors = ButtonDefaults.buttonColors(),
+    contentPadding: PaddingValues = ButtonDefaults.ContentPadding,
+    onClick: () -> Unit,
+    content: @Composable RowScope.() -> Unit = {},
+) {
+    Button(
+        modifier = modifier.heightIn(min = hmrc_button_size_48).fillMaxWidth(), colors = buttonColors,
+        enabled = enabled, shape = shape, contentPadding = contentPadding, onClick = onClick,
+    ) {
+        content.invoke(this)
+        Text(
+            text = text, fontSize = 16.sp, fontWeight = FontWeight.Normal, textAlign = textAlign,
+            modifier = modifier.weight(1f)
+        )
+    }
+}
+
+@Composable
+fun PrimaryButton(
+    text: String,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    textAlign: TextAlign = TextAlign.Center,
+    onClick: () -> Unit,
+) {
+    val primaryButtonColors: ButtonColors = ButtonDefaults.buttonColors(
+        containerColor = colors.hmrcGreen1, contentColor = colors.hmrcWhite
+    )
+    val disabledPrimaryButtonColors: ButtonColors = ButtonDefaults.buttonColors(
+        disabledContainerColor = colors.hmrcGrey2, disabledContentColor = colors.hmrcGrey1
+    )
+    HmrcButton(
+        text = text, modifier = modifier, enabled = enabled, textAlign = textAlign,
+        buttonColors = if (enabled) primaryButtonColors else disabledPrimaryButtonColors,
+        onClick = onClick
+    )
+}
+
+@Composable
+fun SecondaryButton(
+    text: String,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    textAlign: TextAlign = TextAlign.Center,
+    buttonColors: ButtonColors = ButtonDefaults.buttonColors(
+        containerColor = Color.Transparent, contentColor = colors.hmrcBlue
+    ),
+    onClick: () -> Unit,
+    content: @Composable RowScope.() -> Unit = {},
+) {
+    CompositionLocalProvider(LocalRippleTheme provides HmrcRippleTheme) {
+        HmrcButton(
+            text = text, modifier = modifier, contentPadding = PaddingValues(all = hmrc_spacing_16),
```

Comment:
The padding for the current `SecondaryButton` is not al 16dp, can you check if this is appearing the same?

Reply:
to adjust with the buttons we used padding as 16dp, so SecondaryButton also shows the same behaviour as in xml one